yearn/contracts/SaddleStrategy.sol

```diff
@@ -0,0 +1,393 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "@keep-network/yearn-vaults/contracts/BaseStrategy.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+
+/// @notice Interface for the optional metadata functions from the ERC20
+///         standard.
+interface IERC20Metadata {
+    function symbol() external view returns (string memory);
+}
+
+/// @notice Interface for the Saddle Swap contract.
+/// @dev This is an interface with just a few function signatures of the pool.
+///      For more info and function description please see:
+///      https://github.com/saddle-finance/saddle-contract/blob/master/contracts/Swap.sol
+///      https://github.com/saddle-finance/saddle-contract/blob/master/contracts/guarded/SwapGuarded.sol
+interface ISaddlePoolSwap {
+    function addLiquidity(
+        uint256[] calldata amounts,
+        uint256 minToMint,
+        uint256 deadline,
+        bytes32[] calldata merkleProof
+    ) external returns (uint256);
+}
+
+/// @notice Interface for the LPRewardsTBTCSaddle reward pool contract.
```

Comment:
`Interface for the LPRewardsTBTCSaddle reward pool contract.` -> `Interface for the LPRewards contracts.` ?

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/13/commits/0449be392c7e1f92acb63f46b6526b1901c30467.